global.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "3.2.0"
   },
   "sdk": {
-    "version": "6.0.403",
+    "version": "7.0.102",
```

Comment:
Should we just use 7.0.x here?

Reply:
No we need to avoid 101 as there was a very bad bug that was fixed in 102